src/SignalR/clients/ts/signalr/src/HttpConnection.ts

```diff
@@ -147,7 +148,13 @@ export class HttpConnection implements IConnection {
         }
 
         // Transport will not be null if state is connected
-        return this.transport!.send(data);
+        const send = () => this.transport!.send(data);
```

Comment:
Now that this is a delegate you can't safely assume `this.transport` is not null